helm/content-store/templates/configmap.yaml

```diff
@@ -30,6 +30,8 @@ data:
         listen {{ .Values.nginxPort }};
 
         location / {
+          proxy_set_header   Host $host;
```

Comment:
#sadpanda that we have to do this - I'm guessing it's a Rails thing?

Reply:
I got caught with that in Publisher. Basically, some rails app uses the host header to create a redirect link. e.g. publisher redirects to authentication URL.

Reply:
Aha, thanks that's good to know.